src/commands/convertConnections/IConvertConnectionsContext.ts

```diff
@@ -0,0 +1,18 @@
+/*---------------------------------------------------------------------------------------------
+*  Copyright (c) Microsoft Corporation. All rights reserved.
+*  Licensed under the MIT License. See License.md in the project root for license information.
+*--------------------------------------------------------------------------------------------*/
+
+import { type IResourceGroupWizardContext, type Role } from "@microsoft/vscode-azext-azureutils";
+import { type ExecuteActivityContext, type ISubscriptionActionContext } from "@microsoft/vscode-azext-utils";
+import { type SlotTreeItem } from "../../tree/SlotTreeItem";
+import { type Connection } from "./SelectConnectionsStep";
+
+export interface IConvertConnectionsContext extends ExecuteActivityContext, IResourceGroupWizardContext, ISubscriptionActionContext {
+    local: boolean;
```

Comment:
Small nit: Is there a point of having a local and function app property? I feel like it is just more things to keep track of-- if there is no functionapp on the context, can we assume that it is local?